internal/graph/dotgraph.go

```diff
@@ -483,9 +483,16 @@ func escapeAllForDot(in []string) []string {
 	return out
 }
 
-// escapeForDot escapes double quotes and backslashes, and replaces Graphviz's
-// "center" character (\n) with a left-justified character.
+// escapeForDot escapes double quotes and backslashes, and replaces newlines
+// with a left-justified escape (\l).
 // See https://graphviz.org/docs/attr-types/escString/ for more info.
 func escapeForDot(str string) string {
 	return strings.ReplaceAll(strings.ReplaceAll(strings.ReplaceAll(str, `\`, `\\`), `"`, `\"`), "\n", `\l`)
 }
+
+// escapeForDotCentered escapes double quotes and backslashes, and replaces
+// newlines with Graphviz's center escape (\n).
+// See https://graphviz.org/docs/attr-types/escString/ for more info.
+func escapeForDotCentered(str string) string {
+	return strings.ReplaceAll(strings.ReplaceAll(strings.ReplaceAll(str, `\`, `\\`), `"`, `\"`), "\n", `\n`)
```

Comment:
I could pull the common parts of these two functions out into a single shared function, if that seems better.